src/main/webapp/app/course/manage/detail/course-detail-doughnut-chart.component.ts

```diff
@@ -21,6 +21,7 @@ export class CourseDetailDoughnutChartComponent implements OnChanges, OnInit {
     @Input() currentAbsolute: number | undefined;
     @Input() currentMax: number | undefined;
     @Input() course: Course;
+    @Input() showText: string | undefined;
```

Comment:
```suggestion
    @Input() showText?: string;
```
Yuu could change the other Inputs, too. (: